BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-1.0.25-prerelease-00221
\ No newline at end of file
+1.0.25-prerelease-00225-00
```

Comment:
It will after I merge this change. Build tools repo was still using the old version of the buildtools package on the previous official build, and in that one BuildVersionMinor didn't work yet so we were defaulting to 00.